README.md

```diff
@@ -122,6 +129,31 @@ A json representation of all of the open triage tasks will be printed to the ter
 
 The scores of all open triage tasks will be updated. The terminal will show progress.
 
+### Reports
+
+The reporting functionality is currently not very generalized and the backing JQL queries are fairly specific to the FC
+team.
+
+#### VFR Sanity Check
+
+The VFR Sanity Check report can be generated using
+```bash
+$ fcli vfrsanity check
+```
+
+This report will create a Google sheet with two tabs.  The first will order and group all stories by Duration value and
+the second will order and group all stories by Cost of Delay value.
+
+#### User Tasks
+
+The User Tasks report can be generated using
+```bash
+$ fcli reports usertasks
```

Comment:
Both the VFR Sanity Check and the User Tasks are reports.  What if you were to condense them both under the same top level command `reports`.  Therefore you will have something like `fcli reports vfrsanity` and `fcli reports usertasks`.

Reply:
Also, there is a decent bit of code boilerplate that is duplicated between the two parts that could be commonized through shared functions.